app/views/plugins/call_tools/_form.slim

```diff
@@ -1,37 +1,32 @@
 .row.plugin.call-tool.collection-editor data-plugin-id=plugin.id
   .col-md-7
-    = form_for plugin, remote: true, html: { class: 'form-element call-tool-form' } do |f|
+    - name = "plugins_call_tool_#{plugin.id}"
+    = form_for plugin, remote: true, as: name, html: { class: 'form-element one-form' }, data: {type: name }  do |f|
       = render 'plugins/shared/plugin_metadata', f: f
       .form-group
         = f.label(:title, t('plugins.call_tool.title'))
         = f.text_field :title, class: 'form-control'
 
-      .form-group
-        = label_with_tooltip(f, :sound_clip, 'Sound Clip', t('tooltips.call_tool.audio'))
-        = f.file_field :sound_clip, class: 'form-control'
 
-      .sound-clip-detail
-        - if plugin.sound_clip.present?
-          = render partial: 'plugins/call_tools/sound_clip_detail', locals: { plugin: plugin }
-
-      .form-group
-        = label_with_tooltip(f, :targets_csv_file, 'Targets CSV', t('tooltips.call_tool.targets'))
-        = f.file_field :targets_csv_file, class: 'form-control'
-
-      = render partial: 'plugins/call_tools/targets', locals: { targets: plugin.targets }
 
       .form-group
         = label_with_tooltip(f, :description, t('plugins.call_tool.talking_points'), t('tooltips.content'))
         .wysiwyg id='call-tool-wysiwyg'
         = f.hidden_field :description, id: "call-tool-wysiwyg_content"
 
-      .form-group
-        = submit_tag t('common.save'), class: 'btn btn-default xhr-feedback'
+    .well.targets-section
+      = render partial: 'plugins/call_tools/target_form.slim', locals: { plugin: plugin }
```

Comment:
the `.slim` extension should be unneccessary. In fact, from this file, you should be able to just write `render 'target_form'`